Current instructions for backend_failure_recovery.md (#2557)
chef/chef-web-docs: content/backend_failure_recovery.md

However if the failing node cannot be brought back online, it must be
replaced:

> 

Reviewer: ```suggestion
```

1. Run `chef-backend-ctl remove-node NODE_NAME` from any working cluster member to remove the offending node (it doesn't have to be the leader).
1. Run `chef-backend-ctl cleanse` on the offending node.  This will save configuration files under the root directory by default.

> 1. Check to make sure `/var/opt/chef-backend` was deleted by `chef-backend-ctl cleanse`.

Reviewer: ```suggestion
1. Verify that `/var/opt/chef-backend` was deleted in the previous step.
```

> 1. `mkdir /etc/chef-backend` # make a place to move configuration files in a more self-evident place

Reviewer: ```suggestion
1. Make a directory for the configuration files: `mkdir /etc/chef-backend`.
```
